util.go

```diff
@@ -9,3 +23,163 @@ func StringPtr(s string) *string {
 func BoolPtr(b bool) *bool {
 	return &b
 }
+
+// listContainsString returns true if the given list contains the given string.
+func listContainsString(strings []string, targetString string) bool {
+	for _, aString := range strings {
+		if aString == targetString {
+			return true
+		}
+	}
+	return false
+}
+
+// lowestSmartRate returns the lowest smartrate from the given list of smartrates.
+func (c *Client) lowestSmartRate(rates []*SmartRate, deliveryDays int, deliveryAccuracy string) (out SmartRate, err error) {
+	validDeliveryAccuracies := []string{"percentile_50", "percentile_75", "percentile_85", "percentile_90", "percentile_95",
+		"percentile_97", "percentile_99"}
+
+	// if the delivery accuracy is not valid, return an error
+	if !listContainsString(validDeliveryAccuracies, deliveryAccuracy) {
```

Comment:
```suggestion
	if !listContainsString(validDeliveryAccuracies, strings.ToLower(deliveryAccuracy)) {
```

Let's lowercase `deliberyAccuracy` so it will not be case sensitive when checking `deliveryAccuracy` is valid or not.